src/components/AccordionList.tsx

```diff
@@ -0,0 +1,31 @@
+import { useState } from "react";
+import { useTestIds } from "..";
+import { Accordion, AccordionProps, AccordionSize } from "./Accordion";
+
+interface AccordionListProps {
+    accordions: AccordionProps[];
+    /** Allows multiple accordions to be expanded simultaneously (enabled by default) */
+    allowMultipleExpanded?: boolean;
+    size?: AccordionSize;
+}
+
+export function AccordionList({ accordions, size, allowMultipleExpanded = true }: AccordionListProps) {
+    const [currentSelectedIndex, setCurrentSelectedIndex] = useState<number>();
+    const tid = useTestIds({}, "accordionList");
+
+    return (<>
+        {accordions.map((accordionProps, index, arr) => (
+            <Accordion
+                {...accordionProps}
+                {...tid[index]}
```

Comment:
**suggestion** I would prefer we just do `{...tid}` without the index. In our tests we can always get the index if needed either by `r.queryAllByTestId("accordionList_title")[0]`, or our own helper `r.accordionList_title_0`.